### ISSUE TITLE
Alter Vite config, set serverPublic just to dist

### ISSUE THREAD

**Issue by mikewink** [mandrasch/ddev-craftcms-vite#8]
https://github.com/mandrasch/ddev-craftcms-vite/blob/bbd0b3ba38325825ccc18229d1a0068133ef50fd/config/vite.php#L23

I had a problem with multisite installations and CORS with that current setting. Maybe that can be left out? It works for me™. ;)

**Comment by mandrasch** [comment]
Hey @mikewink, 

interesting, thanks very much! I'll have a look.

**Comment by mandrasch** [comment]
Hey @mikewink, short update:

I checked the craft vite docs where I got this setting from. They state

> serverPublic - the public server URL to your asset files; it can be a full URL or a partial path, or a Yii2 alias. This will appear in <script> tags on the frontend for production builds. App::env('PRIMARY_SITE_URL') . '/dist/' is a typical setting
https://nystudio107.com/docs/vite/

My first thought was "Oh, maybe sub-sites like https://ddev-craftcms-vite.ddev.site/de/ won't work then", but `/dist/` is from URL root. So I just checked with https://ddev-craftcms-vite.ddev.site/de/ and it works. 👍 

Need to confirm that everything works with Codespaces as well when I find time.

Thanks very much for your input 👍
